modules/http-api/src/main/java/org/mule/runtime/http/api/sse/SseClient.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+
+/**
+ * Server-side abstraction of a connected client.
+ * <p>
+ * This API is EXPERIMENTAL. Do not use it until it is stable.
+ */
+@Experimental
+@NoImplement
+public interface SseClient extends AutoCloseable {
```

Comment:
rename to `SseRemoteClient`? to better disambiguate it form the actual client. (maybe jsut splitting into subpackages does it)

Reply:
I split the packages between client and server side things